k8s/go/cmd/resolver/resolver.go

```diff
@@ -233,11 +233,11 @@ func resolveString(r *resolver, s string) (string, error) {
 	}
 	auth, err := authn.DefaultKeychain.Resolve(t.Context())
 	if err != nil {
-		return "", fmt.Errorf("unable to get the authenticator using the default keychain for image %v: %v", t, auth)
+		return s, nil
```

Comment:
Can you also include a short explanation in the function description comment why the returned error needs to be always nil.

Reply:
Ack. Will address in follow up PR as prow CI has been running for 10mins already it takes ~30mins